src/jenesis/cmd/attach.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import os
+
+from cosmpy.aerial.client import LedgerClient
+from jenesis.config import Config
+from jenesis.contracts.detect import detect_contracts
+from jenesis.contracts.monkey import MonkeyContract
+from jenesis.contracts.networks import get_network_config
+
+
+def run(args: argparse.Namespace):
+
+    project_path = os.getcwd()
+
+    # check that we are actually running the command from the project root
+    if not os.path.exists(os.path.join(project_path, "jenesis.toml")):
+        # pylint: disable=all
+        print("Please run command from project root")
+        return
+
+    cfg = Config.load(project_path)
+
+    if args.profile not in cfg.profiles:
+        print(f'Invalid profile name. Expected one of {",".join(cfg.profiles.keys())}')
+        return
+
+    selected_profile = cfg.profiles[args.profile]
+    network_cfg = get_network_config(selected_profile.network)
+    if network_cfg is None:
+        print("Not network configuration for this profile")
+        return
+
+    contracts = detect_contracts(project_path)
+
+    for contract in contracts:
+        if contract.name == args.contract:
+            selected_contract = contract
+            continue
+
+    client = LedgerClient(network_cfg)
+
+    contract = MonkeyContract(selected_contract.binary_path, client, args.address)
+    code_id = contract.code_id
+    digest = contract.digest.hex()
+
+    Config.create_project(project_path, args.profile)
```

Comment:
We should maybe rethink how to update a project file without overwriting user changes. This would overwrite any changes a user made, like adding init arguments, right?

Reply:
Yes, I was actually thinking of creating another function called update_project to be used in the attach and add commands, just reading whats already in the jenesis.toml file and adding whats new